src/LiveComponent/assets/package.json

```diff
@@ -17,7 +17,7 @@
     "main": "dist/live_controller.js",
     "types": "dist/live_controller.d.ts",
     "config": {
-        "css_source": "styles/live.css"
+        "css_source": "src/live.css"
```

Comment:
I moved the `live.css` file under `src` aside js source code, otherwise tsdown/rolldown create:
- `dist/src/live_controller.js`
- `dist/styles/live.min.css`

Which is wrong.

Note: other UX packages define their styles in `src/` too